.github/workflows/tests.yaml

```diff
@@ -89,6 +89,8 @@ jobs:
         include:
           - os: ubuntu-latest
             python-version: '3.10'
+          - os: ubuntu-latest
+            python-version: '3.11'
```

Comment:
well python 3.11 is not yet available in Conda 

https://github.com/conda-forge/conda-forge.github.io/issues/1629#issuecomment-1279303846

Reply:
✅